### PR TITLE
go/raindrops: missing comments for disapprove

### PR DESCRIPTION
**Pull request by tehsphinx**

### CODE REVIEW

File: automated-comments/go/raindrops/missing_entry_function.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The `Convert` function is missing or couldn't be found.
```
Comment (ErikSchierboom) [inline automated-comments/go/raindrops/missing_entry_function.md]: What is the difference between missing and couldn't be found? Maybe just do:

```suggestion
The `Convert` function could not be found. Add the `Convert` function, which is required by the tests.
```

File: automated-comments/go/raindrops/entry_signature_changed.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The signature of the `Convert` function should be `Convert(i int) string`.
+Exercism is built on the TDD (Test Driven Development) principle.
```
Comment (ErikSchierboom) [inline automated-comments/go/raindrops/entry_signature_changed.md]: Should we perhaps link to a page that has a basic introduction of TDD?

File: automated-comments/go/raindrops/entry_signature_changed.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+The signature of the `Convert` function should be `Convert(i int) string`.
```
Comment (ErikSchierboom) [inline automated-comments/go/raindrops/entry_signature_changed.md]: Should this perhaps be a general comment, where the function name and the expected signature are parameters? That way, this comment can be re-used in multiple analyzers.

File: automated-comments/go/raindrops/missing_entry_function.md (hunk 1 above)
Comment (ErikSchierboom) [reply automated-comments/go/raindrops/missing_entry_function.md]: Should this perhaps be a general comment, where the function name is passed as a parameter? That way, this comment can be re-used in multiple analyzers.

File: automated-comments/go/raindrops/entry_signature_changed.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+Exercism is built on the TDD (Test Driven Development) principle. 
+The tests already exist and define the outward facing interface as well as the behaviour of the code to be implemented.
+Changing the signature will break the tests.
```
Comment (kytrinyx) [inline automated-comments/go/raindrops/entry_signature_changed.md]:
```suggestion
Changing the signature breaks the tests.
```

File: automated-comments/go/raindrops/entry_signature_changed.md (hunk 2 above)
Comment (tehsphinx) [reply automated-comments/go/raindrops/entry_signature_changed.md]: What do you think about this one?
http://testfirst.org/

It's mostly about Test Driven Teaching. Basically what exercism.io does.

TDD is mostly about writing the tests first which the students don't do here.

Comment (tehsphinx) [reply automated-comments/go/raindrops/entry_signature_changed.md]: Maybe this direct link into the website: http://testfirst.org/about#tdd

Comment (tehsphinx) [reply automated-comments/go/raindrops/entry_signature_changed.md]: Or just Wikipedia? https://en.wikipedia.org/wiki/Test-driven_development

Comment (ErikSchierboom) [reply automated-comments/go/raindrops/entry_signature_changed.md]: Either could work, although I have a slight preference for the http://testfirst.org/about#tdd one, due to its brevity.

File: automated-comments/go/raindrops/entry_signature_changed.md (hunk 3 above)
Comment (SleeplessByte) [inline automated-comments/go/raindrops/entry_signature_changed.md]:
```suggestion
Change the signature of the `Convert` function to be `Convert(i int) string`.
```

Perhaps?

Comment (tehsphinx) [reply automated-comments/go/raindrops/entry_signature_changed.md]: I applied it, but I will switch to the one in `general` soon anyway. There it says:

```
Fix the signature of the `%{function}` function. The expected signature is `%{signature}`.
```

**Comment by tehsphinx** [comment]
@iHiD: Can we get this merged? Even if we might rephrase things again? Currently mentors are missing these comments and then wonder why it says not to approve.

**Comment by iHiD** [comment]
Merging. We can move forward on new PRs.

**Pull request merged**